### PR TITLE
Feature/cursos backend

### PR DESCRIPTION
**Pull request by Ronal2-Gtz**

### CODE REVIEW

File: Backend/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (jotauribe) [inline Backend/index.html:1]: Delete this file

File: Frontend/public/index.html [hunk 2]
```diff
@@ -3,41 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
```
Comment (jotauribe) [inline Frontend/public/index.html:8]: Why did you remove this meta tag?

File: Frontend/public/index.html [hunk 3]
```diff
@@ -3,41 +3,10 @@
...
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (jotauribe) [inline Frontend/public/index.html:17]: Why did you remove this link tag?

File: Frontend/public/index.html [hunk 4]
```diff
@@ -3,41 +3,10 @@
...
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <title>Cursos</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (jotauribe) [inline Frontend/public/index.html:30]: Why did you remove this tag?

File: Frontend/public/manifest.json [hunk 5]
```diff
@@ -1,25 +0,0 @@
-{
```
Comment (jotauribe) [inline Frontend/public/manifest.json:1]: Why did you remove this?

File: Frontend/public/robots.txt [hunk 6]
```diff
@@ -1,3 +0,0 @@
-# https://www.robotstxt.org/robotstxt.html
```
Comment (jotauribe) [inline Frontend/public/robots.txt:1]: Why did you remove this?

File: Frontend/src/App.js [hunk 7]
```diff
@@ -1,26 +1,23 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import Curso from "./components/Curso";
```
Comment (jotauribe) [inline Frontend/src/App.js:4]: Add a space below this line

File: Frontend/src/serviceWorker.js [hunk 8]
```diff
@@ -1,141 +0,0 @@
-// This optional code is used to register a service worker.
```
Comment (jotauribe) [inline Frontend/src/serviceWorker.js:1]: Why did you remove this file?

File: Frontend/src/setupTests.js [hunk 9]
```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
```
Comment (jotauribe) [inline Frontend/src/setupTests.js:5]: Why did you remove this?

File: Frontend/public/index.html [hunk 10]
```diff
@@ -3,41 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (jotauribe) [inline Frontend/public/index.html:6]: Why did you remove this?

File: Frontend/src/components/Curso/HeaderCursos.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
```
Comment (jotauribe) [inline Frontend/src/components/Curso/HeaderCursos.js:1]: Create subfolders for these subcomponents and create separate styles for each one.

File: Frontend/src/components/Curso/FooterCursos.js [hunk 12]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+class Footer extends Component {
+  render() {
+    const { price, rating } = this.props;
+    let estrellas = "";
+
+    for (var i = 0; i < Math.round(rating); i++) {
```
Comment (jotauribe) [inline Frontend/src/components/Curso/FooterCursos.js:8]: Extract the start creation functionality into a function outside of the component

File: Frontend/src/components/Curso/HeaderCursos.js [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+
+class Header extends Component {
```
Comment (jotauribe) [inline Frontend/src/components/Curso/HeaderCursos.js:3]: This should be a functional component

File: Frontend/src/components/Curso/FooterCursos.js [hunk 14]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+class Footer extends Component {
```
Comment (jotauribe) [inline Frontend/src/components/Curso/FooterCursos.js:3]: This should be a functional component

File: Frontend/src/components/Curso/BodyCursos.js [hunk 15]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+
+class Body extends Component {
```
Comment (jotauribe) [inline Frontend/src/components/Curso/BodyCursos.js:3]: This should be a functional component

File: Backend/index.js [hunk 16]
```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const axios = require("axios");
+
+app.use(cors());
+const url =
+  "https://mytablemesa.com/api/courses?orderBy=popularity+desc&expand=provider&limit=24&profession=&subjectAreaCode=&state=&provider=&name=";
```
Comment (jotauribe) [inline Backend/index.js:8]: please use dotenv lib to store this url https://www.npmjs.com/package/dotenv

File: Frontend/src/components/Curso/FooterCursos.js (hunk 12 above)
Comment (Ronal2-Gtz) [reply Frontend/src/components/Curso/FooterCursos.js:8]: Ready

File: Frontend/src/components/Curso/BodyCursos.js (hunk 15 above)
Comment (Ronal2-Gtz) [reply Frontend/src/components/Curso/BodyCursos.js:3]: I will try that

File: Frontend/src/components/Curso/FooterCursos.js (hunk 14 above)
Comment (Ronal2-Gtz) [reply Frontend/src/components/Curso/FooterCursos.js:3]: I will try that

File: Frontend/src/components/Curso/HeaderCursos.js (hunk 13 above)
Comment (Ronal2-Gtz) [reply Frontend/src/components/Curso/HeaderCursos.js:3]: I will try that

File: Frontend/src/App.js (hunk 7 above)
Comment (Ronal2-Gtz) [reply Frontend/src/App.js:4]: Ready

File: Backend/index.html (hunk 1 above)
Comment (Ronal2-Gtz) [reply Backend/index.html:1]: Ready

File: Frontend/public/manifest.json (hunk 5 above)
Comment (Ronal2-Gtz) [reply Frontend/public/manifest.json:1]: Already solved

File: Frontend/public/robots.txt (hunk 6 above)
Comment (Ronal2-Gtz) [reply Frontend/public/robots.txt:1]: Already solved

File: Frontend/src/serviceWorker.js (hunk 8 above)
Comment (Ronal2-Gtz) [reply Frontend/src/serviceWorker.js:1]: Already solved

File: Frontend/src/setupTests.js (hunk 9 above)
Comment (Ronal2-Gtz) [reply Frontend/src/setupTests.js:5]: Already solved

File: Frontend/public/index.html (hunk 10 above)
Comment (Ronal2-Gtz) [reply Frontend/public/index.html:6]: Already solved

**Review by jotauribe — APPROVED** [review]
